README.md

```diff
@@ -360,6 +361,92 @@ snet contract <ContractName> [--at AT] <functionName> PARAM1, PARAM2, ... [--tra
 
 ---
 
+```
+snet service init [--name NAME]
+                  [--model MODEL]
+                  [--organization ORGANIZATION]
+                  [--price PRICE]
+                  [--endpoint ENDPOINT]
+                  [--tags TAGS [TAG1, TAG2, ...]]
+                  [--description DESCRIPTION]
+                  [-y]
+```
+
+* Create a service.json file in the current directory either interactively or by passing command line arguments
+* `NAME`: name of the service to be stored in the registry
+* `MODEL`: local filesystem path to the service model directory
+* `ORGANIZATION`: the organization to which you want to register the service
+* `PRICE`: initial price for interacting with the service
+* `ENDPOINT`: initial endpoint to call the service's API 
+* `TAGS`: tags to describe the service
+* `DESCRIPTION`: human-readable description of the service 
+* `-y`: accept defaults for any argument that is not provided
+
+---
+
+```
+snet service publish [NETWORK] [--no-register]
```

Comment:
make `NETWORK` metavar/arg name consistent between `service publish`, `service update`, and `network` in both documentation + arguments.py (is currently `NETWORK_NAME` for network command and `NETWORK` for service subcommands)

Reply:
Changed to NETWORK for all of them since that's what the help text says